x-pack/test_serverless/README.md

```diff
@@ -133,3 +133,21 @@ node scripts/functional_tests_server.js --config test_serverless/api_integration
 
 node scripts/functional_test_runner.js --config test_serverless/api_integration/test_suites/search/config.ts
 ```
+
+## Skipping tests for MKI run
```

Comment:
I'd suggest to change order slightly here and move this section after `Run tests on MKI`.